source/errors.rst

```diff
@@ -35,7 +35,7 @@ Slug) which does not match an object in the database.
 
   {
     "status": 404,
-    "error": "Object not found",
+    "error": "Object Not Found",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L33 shows `'Object not found'` is the correct form.